apps/src/templates/teacherDashboard/sections.js

```diff
@@ -79,25 +91,13 @@ export function renderSectionTable(sectionId, loginType) {
     url: dataUrl,
     dataType: 'json'
   }).done(studentData => {
-    studentData = studentData.map((student) => {
-      return {
-        id: student.id,
-        name: student.name,
-        username: student.username,
-        age: student.age,
-        gender: student.gender,
-        secretWords: student.secret_words,
-        secretPicturePath: student.secret_picture_path,
-        sectionId: sectionId,
-        loginType: loginType,
-      };
-    });
+    store.dispatch(
```

Comment:
I imagine the get request might get moved to redux if I need to have some sort of waiting UI.